### PR TITLE
use ckanext-switzerland-solr-schema

### PR DESCRIPTION
**Pull request by stefina**
this version is a copy of the solrconfig.xml that we use in our SOLR-image

### CODE REVIEW

File: bin/travis-build.bash [hunk 1]
```diff
@@ -5,7 +5,6 @@ echo "This is travis-build.bash..."
 
 echo "Installing the packages that CKAN requires..."
 sudo apt-get update -qq
-sudo apt-get install solr-jetty libcommons-fileupload-java
```
Comment (stefina) [inline bin/travis-build.bash:8]: removing this, as we no longer use jetty for SOLR

File: bin/travis-build.bash [hunk 2]
```diff
@@ -33,9 +32,9 @@ pip install -r dev-requirements.txt
 cd -
 
 echo "Setting up Solr..."
-printf "NO_START=0\nJETTY_HOST=127.0.0.1\nJETTY_PORT=8983\nJAVA_HOME=$JAVA_HOME" | sudo tee /etc/default/jetty
-sudo cp ckan/ckan/config/solr/schema.xml /etc/solr/conf/schema.xml
-sudo service jetty restart
+docker run --name ckan_solr -p 8983:8983 -e SOLR_HEAP=1024m -v "$PWD/solr:/var/solr" -d solr:6 solr-precreate ckan
+docker exec ckan_solr bash -c "cp /var/solr/* /opt/solr/server/solr/mycores/ckan/conf/"
```
Comment (stefina) [inline bin/travis-build.bash:36]: the core is created when running the ckan_solr-container. that folder already contains some conf-files that we do not particularly add so we use the default and add our custom files from the mounted `/var/solr`-dir (that originates from the `solr`-folder of this repo)

**Comment by stefina** [comment]
The conf-files that are added to the `solr`-file contain the schema.xml and other config. Please review the schema.xml, if it is the right version. The other files should just be a copy of the original root-repo which will be removed in the upper PR.

File: solr/schema.xml [hunk 3]
```diff
@@ -261,6 +261,7 @@ schema. In this case the version should be set to the next CKAN version number.
     <field name="publishers" type="textgen" indexed="true" stored="true" multiValued="true"/>
     <field name="identifier" type="string" indexed="true" stored="true"/>
     <field name="see_alsos" type="string" indexed="true" stored="true" multiValued="true"/>
+    <field name="private" type="boolean" indexed="true" stored="true" multiValued="false"/>
```
Comment (stefina) [inline solr/schema.xml:264]: Is this line added correctly?

Comment (bellisk) [reply solr/schema.xml:264]: Thanks for pointing that out! Yes, it's correct. It's the field we need so we can facet on private/published datasets (e.g. draft/published).

**Pull request merged**